examples/Examples/Rationals.hs

```diff
@@ -0,0 +1,65 @@
+module Examples.Rationals (tests) where
+
+import Test.Tasty
+import Test.Tasty.HUnit
+
+import Utils
+
+import Plutarch
+import Plutarch.Bool
+import Plutarch.Builtin
+import Plutarch.Pair
+import Plutarch.Rational
+
+--import Data.Ratio ((%))
+
+tests :: HasTester => TestTree
+tests = do
+  testGroup
+    "rational tests"
+    [ testCase "1/2 + 1/2 = 1" $
+        expect $ 1 / 2 + 1 / 2 #== (1 :: Term s PRational)
+    , testCase "1/2 - 1/3 = 1/6" $
+        expect $ 1 / 2 - 1 / 3 #== (1 / 6 :: Term s PRational)
+    , testCase "2/9 < 3/10" $
+        expect $ 2 / 9 #< (3 / 10 :: Term s PRational)
+    , testCase "harmonic sum" $
+        expect $ 1 / 2 + 1 / 3 + 1 / 4 + 1 / 5 #== (77 / 60 :: Term s PRational)
+    , testCase "product" $
+        expect $ 1 / 2 * 2 / 3 * 3 / 4 * 4 / 5 * 5 / 6 #== (1 / 6 :: Term s PRational)
+    , testCase "round 5/3" $
+        expect $ pround # (5 / 3) #== 2
+    , testCase "round 4/3" $
+        expect $ pround # (4 / 3) #== 1
+    , testCase "round 5/2" $
+        expect $ pround # (5 / 2) #== 3
+    , testCase "round 9/4" $
+        expect $ pround # (9 / 4) #== 2
+    , testCase "round 11/4" $
+        expect $ pround # (11 / 4) #== 3
```

Comment:
We need to cover the 'equidistant' case described above. There are also no tests dealing with negative values.